CHANGELOG.md

```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.13.0]
-### Uncategorized
+### Added
+- **[FLASK]** Add Snaps settting to search index ([#14100](https://github.com/MetaMask/metamask-extension/pull/14100))
+- Add token list aggregattor to token details page ([#14051](https://github.com/MetaMask/metamask-extension/pull/14051))
```

Comment:
"Add token list aggregattor to token details page" this is still behind feature flag, and so should not be in the changelog yet